org.civicrm.bootstrapcivihr/scss/components/_spinner.scss

```diff
@@ -0,0 +1,3 @@
+.spinner {
+  @include spinner(1s);
```

Comment:
if it's always `1s` both here and in the override, then just put it as a default value in the mixin